{"name": "Курьер авторизирован под несуществующим логином", "status": "passed", "description": "Проверка авторизации курьера в системе под несуществующим пользователем", "start": 1727813106364, "stop": 1727813106444, "uuid": "8341ba7d-d3d8-4f4d-8315-3ae9b8ec6442", "historyId": "ede39be2e6b88a3d6dc88c6bd4b3e910", "testCaseId": "ede39be2e6b88a3d6dc88c6bd4b3e910", "fullName": "tests.test_courier_login.TestCourierLogin#test_check_authorization_under_incorrect_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-AMN9CM3"}, {"name": "thread", "value": "17916-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}